pubnub/models/subscription.py

```diff
@@ -73,7 +71,11 @@ def signal(self, signal):
 
 
 class PNSubscribeCapable:
-    def subscribe(self):
+    def subscribe(self, timetoken: Optional[int] = None, region: Optional[str] = None,
+                  with_presence: Optional[bool] = None):
```

Comment:
`with_presence` should be specified when _creating_ the subscription (`channel.subscription()`), not in `subscribe()`

Reply:
should this still be here? it's both in `subscription()` and `subscribe()` now, and according to EL spec it should be in `subscription()` only

Reply:
This is a small detour I had to make to keep it reverse compatible to be honest.